types/src/v0/impls/auction.rs

```diff
@@ -29,11 +29,24 @@ impl FullNetworkTx {
     }
 }
 
-impl Committable for BidTxBody {
+impl Committable for BidTx {
     fn tag() -> String {
         "BID_TX".to_string()
     }
 
+    fn commit(&self) -> Commitment<Self> {
+        let comm = committable::RawCommitmentBuilder::new(&Self::tag())
+            .field("body", self.body.commit())
```

Comment:
I think it was probably a previous mistake to have an explicit commitment for `BidTxBody`. The `BidTx` can just access `body` methods (or fields) directly in its own commitment. Having a `body` is a structural necessity but behavior should act as if all the fields are on `BidTx`.